Unity/NativeObject.V1.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Specialized;
+
+namespace Unity
+{
+    public partial class NativeObject
+    {
+        internal unsafe class V1 : INativeObjectImpl
+        {
+            NativeObject* nativeObject;
+
+            public int InstanceID => nativeObject->InstanceID;
+
+            public BitVector32 Bits => nativeObject->bits;
+
+            public IntPtr Pointer => new IntPtr(nativeObject);
+
+            public V1(IntPtr ptr)
+            {
+                nativeObject = (NativeObject*)ptr;
+            }
+
+            internal struct NativeObject
+            {
+                public IntPtr* VirtualFunctionTable { get; }
+                public int InstanceID { get; }
+                public BitVector32 bits { get; }
```

Comment:
These should be fields and not properties, since the order of the underlying fields of auto properties is undefined.